Makefile

```diff
@@ -141,6 +141,7 @@ manually_upload_certs: ## Manually upload to AWS the SSL certificates for public
 .PHONY: pingdom
 pingdom: ## Use custom Terraform provider to set up Pingdom check
 	$(eval export PASSWORD_STORE_DIR?=~/.paas-pass)
+	$(eval export PINGDOM_CONTACT_IDS=11089310)
```

Comment:
Can we say in the commit message which contact is this? or add the link https://my.pingdom.com/contacts#contact=11089310